vitals/BatteryManagement.java

```diff
@@ -0,0 +1,51 @@
+package vitals;
+
+
+public class BatteryManagement {
+    static boolean batteryIsOk(float temperature, float soc, float chargeRate) {
+    		return (temperatureCheck(temperature) && chargeStateCheck(soc) && chargeRateCheck(chargeRate));    	   	
+    }
+
+    static boolean temperatureCheck(float temperature) {

```

Comment:
Separate I/O operation(System.out.println) from check functions to its own method